modules/_labs/color-picker/react/spec/ColorPicker.spec.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import {fireEvent, render} from '@testing-library/react';
+import {colors} from '@workday/canvas-kit-react-core';
+import {ColorPicker, ColorPickerProps} from '@workday/canvas-kit-labs-react-color-picker';
+
+describe('<ColorPicker /> component', () => {
+  const dataTestId = 'TESTY_TEST';
+
+  const renderColorPicker = (props?: Partial<ColorPickerProps>) => {
+    return render(<ColorPicker data-test-id={dataTestId} onChange={jest.fn()} {...props} />);
+  };
+
+  it('should include aria-label', () => {
+    const ariaLabel = 'Background Color';
+    const {container, unmount} = renderColorPicker({ariaLabel});
+
+    expect(container.querySelector(`[aria-label="${ariaLabel}"]`)).not.toBeNull();
+    unmount();
+  });
+
+  it('should call onChange handler when clicking a color swatch', () => {
+    const onChange = jest.fn();
+    const {container, unmount} = renderColorPicker({onChange});
+    const colorSwatch = container.querySelector('#canvas-color-picker--color-333333')!;
```

Comment:
I think the exclamation point was a typo

```suggestion
    const colorSwatch = container.querySelector('#canvas-color-picker--color-333333');
```

Reply:
Usually selectors like this means we haven't made our component accessible. There is no `aria-label` or anything else to tell a user of an assistive technology which value the picker will get. We can either label with our color names or the color value itself. This would allow us to use `getByLabelText('333333')` instead of putting `id`s in for the sole purpose of selecting in tests.

Reply:
non-null assertion

Reply:
I see why the exclamation is there. `fireEvent.click` probably expects an element and not `null`

Reply:
Another reason to use `getByLabelText`. It will either return an element or throw an error. Either way, getting past that line means an element is returned